eng/templates/Azure.ResourceManager.Template/src/autorest.md

```diff
@@ -29,21 +29,22 @@ rename-rules:
   CPUs: Cpus
```

Comment:
Yeah... unfortunately yes

Reply:
We should file an issue to see if there is some way we can avoid this duplication. I thought autorest config supported importing of other files and if that is the case perhaps we could put these into a common file and just import it.

Reply:
I have to say this is on purpose because changing this will introduce changes in the generated code and usually breaking changes. I could change all of these in bulk because these RPs are not GA yet and we have an update on that and want to apply to all the RPs.
The point is that we maintain the list in the "template" and every RP should be onboard through that template therefore this piece of configuration could apply to the new RPs.
autorest config does support importing of other files, but only for the entries that supported by autorest itself. This configuration is defined in our generator. In addition, it does not support merge configurations - this piece of configuration is not the same in all RPs (they all have a shared part but might have some additional ones)
Duplicating them everywhere is ugly but it works and is flexible. What I am doing in this PR will not happen again in the future.

Reply:
I opened an issue.